ReplayKitExample/BroadcastExtension/SampleHandler.swift

```diff
@@ -125,7 +134,13 @@ class SampleHandler: RPBroadcastSampleHandler {
     override func processSampleBuffer(_ sampleBuffer: CMSampleBuffer, with sampleBufferType: RPSampleBufferType) {
         switch sampleBufferType {
         case RPSampleBufferType.video:
-            videoSource?.processFrame(sampleBuffer: sampleBuffer)
+            if SampleHandler.isScreencast,
+                kFrameCounter % 2 == 0 {
+                videoSource?.processFrame(sampleBuffer: sampleBuffer)
+            } else if !SampleHandler.isScreencast {
+                videoSource?.processFrame(sampleBuffer: sampleBuffer)
+            }
+            kFrameCounter += 1
```

Comment:
The reason for this frame skipping logic is that the CoreImagePixelBufferInput processes every frame, even if WebRTC might drop it due to a format request. On an extension this is can be particularly problematic and skipping frames helps prevent the device overload case where Apple maps 6-8 IOSurfaces into the extension.